features/view_specs/view_spec.feature

```diff
@@ -1,6 +1,55 @@
 Feature: view spec
 
-  View specs live in spec/views and render view templates in isolation.
+  View specs should be placed in `spec/views`,
+  and verify the content of view templates in isolation
+  (that is, without invoking a controller).
+
+  Overview
+  --------
+
+  View specs generally follow three steps:
+
+  ```ruby
+  assign(:widget, Widget.new)  # sets @widget = Widget.new in the view template
+
+  render
+
+  expect(rendered).to match(/text/)
+  ```
+
+  1. Use the `assign` method to set instance variables in the view.
+     Technically, `@widget = Widget.new` would work too,
+     but RSpec doesn't officially support this pattern.
+     (It only works as a side effect of the fact that
+     view specs include `ActionView::TestCase` behavior.
+     Be aware that it may be made unavailable in the future.)
+
+  2. Use the `render` method to render the view.
+
+  3. Set expectations against the resulting `rendered` object.
```

Comment:
Don't enclose `rendered` in back ticks here, its not code. Maybe "rendered" if you want to distinguish it. Also it's not an object but a template. Its actually several objects and not one a developer can access.

Reply:
TIL.